bootstrap/httpserver/server.go

```diff
@@ -6,10 +6,10 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/orbs-network/orbs-network-go/instrumentation"
+	"github.com/orbs-network/membuffers/go"
+	. "github.com/orbs-network/orbs-network-go/instrumentation"
```

Comment:
Please do not import everything from a package, it's bad practice
The only place we allow it is when we import from gomega and ginkgo (and we plan on stop using both)